src/std/10-std-builder.md

```diff
@@ -24,7 +24,7 @@ let sys = import('std/sys');
 
 ### new
 ```
-builder.new() -> builder
+builder.new() -> builder_t_t
```

Comment:
`_t_t` 😂

Reply:
:man_facepalming: Damn, seems I didn't turn on the 'whole word' option when doing the search and replace...